### PR TITLE
New package: Converseen-0.9.8.1

### PR DESCRIPTION
**Pull request by uninhm**
WARNING: Cross compilation won't work

### CODE REVIEW

**Comment by Chocimier** [comment]
Use cmake build_style.

File: srcpkgs/Converseen/template [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Template file for 'Converseen'
+pkgname=Converseen
+version=0.9.8.1
+revision=1
+archs="i686* x86_64*"
```
Comment (Chocimier) [inline srcpkgs/Converseen/template:5]: now with cmake style will compile for all archs when added some packages to hostmakedepends, see build log

File: srcpkgs/Converseen/template [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+build_style=cmake
+configure_args="-DCMAKE_INSTALL_PREFIX=/usr"
```
Comment (Chocimier) [inline srcpkgs/Converseen/template:7]: this is default with build_style=cmake

File: srcpkgs/Converseen/template [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+hostmakedepends="pkg-config"
+makedepends="qt5-devel qt5-tools-devel ImageMagick libmagick-devel"
+depends="qt5 ImageMagick ghostscript libheif libraw librsvg libwebp libwmf openexr hicolor-icon-theme"
```
Comment (Chocimier) [inline srcpkgs/Converseen/template:10]: Required parts of qt5,  ImageMagick are detected, see generate-runtime-deps in build log. Other libs aren't used directly, but through ImageMagick, remove.

**Comment by Chocimier** [comment]
Also please leave comment when changes are ready to review.

**Comment by unihernandez22** [comment]
It doesn't compile for other archs... What packages should I add to hostmakedepends?

**Comment by ericonr** [comment]
Can you show error messages during the cross build?

**Comment by sgn** [comment]
Fix for cross:

```diff
diff --git a/srcpkgs/Converseen/template b/srcpkgs/Converseen/template
index 38eddb897c..aaa4289c70 100644
--- a/srcpkgs/Converseen/template
+++ b/srcpkgs/Converseen/template
@@ -3,9 +3,9 @@ pkgname=Converseen
 version=0.9.8.1
 revision=1
 build_style=cmake
-hostmakedepends="pkg-config"
+hostmakedepends="pkg-config qt5-host-tools qt5-qmake"
 makedepends="qt5-devel qt5-tools-devel ImageMagick libmagick-devel"
-depends="qt5 ImageMagick ghostscript libheif libraw librsvg libwebp libwmf openexr hicolor-icon-theme"
+depends="ImageMagick ghostscript openexr hicolor-icon-theme"
 short_desc="Free batch image processor"
 maintainer="Unai Hernández <unihernandez22@gmail.com>"
 license="GPL-3.0-or-later"
```

**Comment by Veganizer** [comment]
Had refactor and updatet it in: #29987

**Comment by Chocimier** [comment]
Will continue there.

**Pull request closed without merging**